src/main/kotlin/no/nav/familie/ks/sak/kjerne/logg/LoggService.kt

```diff
@@ -113,4 +114,55 @@ class LoggService(
             )
         )
     }
+
+    fun opprettSettPåVentLogg(behandling: Behandling, årsak: String) {
+        lagreLogg(
+            Logg(
+                behandlingId = behandling.id,
+                type = LoggType.BEHANDLIG_SATT_PÅ_VENT,
+                rolle = SikkerhetContext.hentRolletilgangFraSikkerhetscontext(
+                    rolleConfig,
+                    BehandlerRolle.SAKSBEHANDLER
+                ),
+                tekst = "Årsak: $årsak"
+            )
+        )
+    }
+
+    fun opprettOppdaterVentingLogg(behandling: Behandling, endretÅrsak: String?, endretFrist: LocalDate?) {
+        val tekst = if (endretFrist != null && endretÅrsak != null) {
+            "Frist og årsak er endret til $endretÅrsak og ${endretFrist.tilKortString()}"
+        } else if (endretFrist != null) {
+            "Frist er endret til ${endretFrist.tilKortString()}"
+        } else if (endretÅrsak != null) {
+            "Årsak er endret til $endretÅrsak"
+        } else {
+            return
+        }
```

Comment:
```suggestion
        val tekst = when {
            endretFrist != null && endretÅrsak != null -> "Frist og årsak er endret til $endretÅrsak og ${endretFrist.tilKortString()}"
            endretFrist != null -> "Frist er endret til ${endretFrist.tilKortString()}"
            endretÅrsak != null != null -> "Årsak er endret til $endretÅrsak"
            else -> {
                logger.info("Oppretter ikke logg alikevel fordi det var ingen endringer?")
                return
            }
        }
```

Lurer på om det hadde vært litt mer leselig med en when (ikke et krav altså 😛)
Skulle man forresten logge noe hvis det ikke er noen endringer, slik at man vet at man ikke har OppdaterVentingLogg? Akkurat nå så returner den bare uten å si ifra hva som er resultatet av  opprettOppdaterVentingLogg

Reply:
Høres lurt ut! 👍 Legger det inn!